tests/examples_arguments_syntax/line_chart_with_cumsum_faceted.py

```diff
@@ -0,0 +1,36 @@
+"""
+Faceted Line Chart with Cumulative Sum
+------------------------------
+This chart creates multiple line subcharts from the cumulative sum of a field, one for each category.
+"""
+# category: line charts
+import altair as alt
+from vega_datasets import data
+
+source = data.disasters()
+columns_sorted = ['Drought', 'Epidemic', 'Earthquake', 'Flood']
+
+alt.Chart(source).mark_line(
+    interpolate='basis' # Use linear interpolation with B-spline method
+).encode( 
+    alt.X('Year:Q', title=None, axis=alt.Axis(format='d')),
+    alt.Y('cumulative:Q', title=None),
+    alt.Color('Entity:N', legend=None)
+).properties(width=300, height=150).facet(
```

Comment:
```suggestion
).properties(
    width=300,
    height=150
).facet(
```

Same in the other file

Reply:
- Revised formatting of chart properties